topics/variant-analysis/tutorials/somatic-variant-discovery/workflows/Control-FREEC_MIRACUM_-_somatic_variant_discovery_from_WES_data.ga

```diff
@@ -0,0 +1,1402 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",

```

Comment:
I see the following errors for the workflow, could you add tags and an annotation for it?

```
Warning:  The tags attribute is missing. Please add: `"tags": ["variant-analysis"]`
Warning:  The 'annotation' attribute is missing or incorrect. Please add: `"annotation": "<title of tutorial>"`
make: *** [Makefile:124: check-workflows] Error 123
```